test/e2e/node/runtimeclass.go

```diff
@@ -102,7 +101,7 @@ var _ = SIGDescribe("RuntimeClass [Feature:RuntimeClass]", func() {
 			err = crds.Delete(runtimeClassCRDName, nil)
 			framework.ExpectNoError(err, "failed to delete RuntimeClass CRD %s", runtimeClassCRDName)
 
-			By("Waiting for the CRD to disappear")
+			By(fmt.Sprintf("Waiting for the CRD (%s) to disappear", string(runtimeClassCRDUID)))
```

Comment:
`Sprintf` is a number of times slower than `Itoa`, please use that

Reply:
Fixed it. Thanks!

Reply:
`runtimeClassCRDUID` is a not an integer (it is [type UID string](https://github.com/kubernetes/apimachinery/blob/095b9d2034671175d5f4d8c61040516f2ce9d05e/pkg/types/uid.go#L22
)), so `Itoa` does not work.